app/src/main/res/layout/list_item_restore_set.xml

```diff
@@ -11,7 +12,6 @@
         android:id="@+id/imageView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="16dp"
```

Comment:
removing this margin here causes the ripple affect to have a sharp border with the icon.

Reply:
will look into this